app/src/main/res/layout/fragment_groups.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".grouping.GroupsFragment">
```

Comment:
nit: Slight formatting improvement. We do these newlines to prevent merge conflicts later on down the road.

```suggestion
    tools:context=".grouping.GroupsFragment"
    >
```